Romuashi/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  if str.length.even?
+    str[str.length / 2 - 1] + str[str.length / 2]
```

Comment:
you use `str.length / 2` several times, probably it's worth to move it local variable